build/networking/scripts/build_p4c.sh

```diff
@@ -32,6 +32,7 @@ echo ""
 git clone https://github.com/p4lang/p4c.git --recursive P4C
 cd P4C
 git checkout $P4C_SHA
+git apply ${WORKDIR}/patches/ipdk_p4c_001.patch
```

Comment:
Can we add an explanation as to why we need the patch and this will be removed once proposal in PNA arch spec is accepted?